pkg/util/errutil/doc.go

```diff
@@ -0,0 +1,44 @@
+// Package errutil provides utilities for working with errors in Grafana.
```

Comment:
I wish more packages had a file like this!

Reply:
🙏 Yeah, I think it's overkill to do it for a lot of packages, but have sort of an ambition to make sure we have at least decent godocs for the infra/util packages going forward 🙂 There's just too many contributors to communicate via osmosis, and pkg.go.dev is sort of where you'd expect this information to live for external packages, so why not for our internal packages as well 🤔